apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Tests/LruCacheTest.cs

```diff
@@ -100,9 +100,9 @@ public void TestWithCapacity100()
         [Fact]
         public void MultithreadedUse()
         {
-            const int threadCount = 50;
-            const int lruCapacity = 1000;
-            const int perThread = 10000;
+            const int threadCount = 5;
+            const int lruCapacity = 100;
+            const int perThread = 1000;
```

Comment:
I'd suggest 10, 1000 and 50000 or 5, 1000 and 100000... I think we want to do the same amount of work, just with fewer threads. (As I believe the problem is lock contention.)